README.md

```diff
@@ -39,14 +39,16 @@ $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\
 ```
 
 ### Restart shell
+If using WSL the following won't work. Close and reopen the WSL window instead.
 ```
 $ exec "$SHELL"
 ```
 
-### Install proper python version
+### Install proper python versions
 ```
+$ pyenv install 2.7.15
```

Comment:
fyi this was to get python 2 installed in WSL. Figured it was okay here, but could see the argument for pulling it out to a WSL only section

Reply:
I see, maybe just add a note about WSL? I think its fine here tho